### PR TITLE
chore: Normalize formatting of the JSON file

### PR DESCRIPTION
**Pull request by blaine-arcjet**
While working on some other PRs, I noticed that VSCode was trying to reformat a lot of this file. This is the format that it really wants to apply.

I think the comma on separate lines was supposed to make adding new entries easier but it was lost at some point. If those should remain, let me know and I can try converting the items that don't adhere to that style; however, I figured following a common editor formatting would make for less churn in future PRs.

### CODE REVIEW

**Comment by monperrus** [comment]
thanks for the PR, if we do that, we also need to modify the validation script in CI.

when it fails, it would give a clear error message "format is wrong, it should be XXXX".

**Comment by blaine-arcjet** [comment]
Sounds good! I'll figure out how to check that in python

**Comment by blaine-arcjet** [comment]
Since there are validations for python, go, and php that just validate the schema, I think this make sense as a separate validation. I'm choosing to write it in JS and run via Node because VSCode uses `JSON.stringify` for the formatting.

I'm also supporting a `--generate` flag on the script to regenerate the file with proper formatting.

File: format.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const fs = require("fs");
```
Comment (monperrus) [inline format.js:1]: add some documentation at the top of the file to say what it does

File: format.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+const updated = JSON.stringify(JSON.parse(original), null, 2) + '\n';
+
+if (process.argv[2] === "--generate") {
+    fs.writeFileSync(jsonFilePath, updated);
+    process.exit(0);
+}
+
+if (process.argv[2] === "--check") {
+    if (updated !== original) {
+        console.error("JSON file format is wrong. Run `node format.js --generate` to update.");
```
Comment (monperrus) [inline format.js:17]: explain the expected format (indentation type, comma convention)

**Review by monperrus — CHANGES_REQUESTED** [review]
LGTM, ok to do this in JS.

We improve documentation and we merge.

File: format.js (hunk 2 above)
Comment (blaine-arcjet) [reply format.js:29]: I made it an error log describing the format

**Pull request merged**